app/controllers/api/v1/reservations_controller.rb

```diff
@@ -0,0 +1,64 @@
+class Api::V1::ReservationsController < ApplicationController
+  before_action :set_reservation_params, only: %i[show update destroy]
+  # before_action :logged_in, except: %i[index show]
+  # before_action :user_ability, except: %i[update destroy]
```

Comment:
- [ ] Please, remove these old pieces of commented code to improve the cleanliness of the repo 💪

Reply:
Resolved, thanks.